docs/docs/plugins-themes-and-starters.md

```diff
@@ -36,3 +36,83 @@ Gatsby offers [official starters](/docs/starters/#official-starters) for a defau
 - [Starter library](/starters/)
 - [Creating a starter](/docs/creating-a-starter/)
 - [Converting a starter to a theme](/docs/themes/converting-a-starter/)
+
+## Conventions for usage
+
+Themes are a type of plugin, making both themes and plugins capable of the same functionality. The difference between them is _intended usage_. Themes are intended to own a piece of the site (like an about us page), whereas a plugin is intended to modularize Gatsby APIs into smaller pieces. Themes tend to cover a broader scope of responsibility, packaging up multiple behaviors, where plugins are meant for more specific functionality.
+
+Starters are generally used as the starting point that plugins and themes are then installed into. However, they're based on one-time use and do not get updated over time as plugins and themes do.
+
+## Comparing differences
+
+Once you have a conceptual understanding of what a plugin, theme, and starter is, you may ask yourself what each is suited for in relation to each other.
+
+The following tables put plugins, themes, and starters side-by-side, showing what each is more appropriate for.
+
+**Legend**
+
+| Icon | Feature Capability                                               |
+| ---- | ---------------------------------------------------------------- |
+| ●    | Fully capable (possible and supported)                           |
+| ◐    | Somewhat capable (support is minimal or it is not best practice) |
+| ○    | Not capable                                                      |
+
+### Differences and considerations in maintenance
+
+When it comes to maintaining a Gatsby site, plugins and themes offer a distinct advantage to starters by being distributed as packages. This means making changes in multiple Gatsby sites is as easy as a new install of an updated package upstream. It's difficult to sync changes across multiple sites derived from the same starter.
+
+| Maintenance        | Plugin | Theme | Starter |
+| ------------------ | ------ | ----- | ------- |
+| Versioning         | ●      | ●     | ◐       |
+| Install as Package | ●      | ●     | ○       |
+
+#### Versioning
+
+Starters can still be versioned inside of a repository so that you can track issues or bugs associated with specific updates, but they aren't formally released and published, so they aren't versioned like a plugin or theme is.
+
+#### Installing as a package
+
+Starters can't be installed into existing sites, this problem was one of the motivating factors in developing the newer concept of themes. You can read more about the rationale for themes in the [What are Gatsby Themes guide](/docs/themes/what-are-gatsby-themes/#gatsby-starters).
```

Comment:
_Was_ this a problem? Doesn't seem like they were ever designed to work that way.

```suggestion
Starters can't be installed into existing sites, this limitation was one of the motivating factors in developing the newer concept of themes. You can read more about the rationale for themes in the [What are Gatsby Themes guide](/docs/themes/what-are-gatsby-themes/#gatsby-starters).
```